lib/ansible/modules/yum_repository.py

```diff
@@ -614,6 +631,69 @@ def main():
     if module.params['params']:
         module.fail_json(msg="The params option to yum_repository was removed in Ansible 2.5 since it circumvents Ansible's option handling")
 
+    if module.params['name'] is None and module.params['copr'] is None:
+        module.fail_json(msg="Either the 'name' or the 'copr' parameter needs to be set")
+
+    if module.params['copr']:
```

Comment:
Maybe something like:
```python
if module.params['alwayscheck']:
    # make network request
else:
    # just template with latest URI
    # log.debug("Note: the COPR URI scheme may change in the future")
```

Is there an ansible convention for a parameter like `alwayscheck` or `alwayscheckcoprremote`?

Reply:
(In order to optionally not make frequent, redundant, probably unnecessary requests to the COPR API)